pyveda/vedaset/stream/vedastream.py

```diff
@@ -101,6 +102,15 @@ def batch_iter(self, batch_size):
                 batch.append(self.__next__())
             yield batch
 
+    def batch_generator(self, batch_size, shuffle, **kwargs):
```

Comment:
can we default shuffle to True/False instead of making it a required arg here?

Reply:
changed for vedabase and vedastream